gradle/tools/detekt/detekt-config.yml

```diff
@@ -116,6 +116,7 @@ complexity:
   LongMethod:
     active: true
     threshold: 60
+    ignoreAnnotated: [ 'Composable' ]
```

Comment:
little change to detekt-config rules since I've been fighting this rule for no reason on Composable methods for a while now.

I guess it makes sense most of the time but for composable we can ignore since things can get bigger and sometimes breaking into smaller methods might not be the best